docs: update Flex robot and instrument cleaning instructions

jwwojak opened the pull request:
# Overview

These changes prompted by customer misunderstanding calibration "recommendations" as "requirements." Trying to correct that in the cleaning instructions. 

Fixing any other code oddities. This one is simple and a low priority on the list. Getting to the end of the items.

See RTC-647.

## Test Plan and Hands on Testing

Check for code accuracy and grammar.

## Changelog

Text changes to put emphasis on calibration as optional but recommended when re-attaching an instrument for cleaning. A "nice to have" but not required tone.

## Review requests

n/a

## Risk assessment

Low, documentation only

jwwojak committed:
Update cleaning instructions

- Make sure calibration info is presented as "optional but recommended."

- remove the awkward "rinse wipedown" phrasing in favor of "Rinse off any remaining residue using a cloth dampened with distilled water"

- Fixed a mixed HTML and markdown table.

ecormany commented on docs/flex-manual/docs/maintenance-service.md:
```diff
@@ -65,21 +65,23 @@ To clean the waste chute:
 1. Remove the pipette from the gantry.
-2. Dampen a soft, clean cloth or paper towel with a cleaning solution.
-3. Gently wipe down the following parts:
+1. Dampen a soft, clean cloth or paper towel with a cleaning solution.
```
We don't have to make the change here, but the process of reviewing this change makes me want to _not_ use lazy list numbering in our markdown source. From [the original description of Markdown](https://daringfireball.net/projects/markdown/):
> The overriding design goal for Markdown’s formatting syntax is to make it as readable as possible. The idea is that a Markdown-formatted document should be publishable as-is, as plain text

So here, if I'm not especially worried about the output formatting, I should be able to read these steps in the GitHub diff, and be able to easily see which is, say, step 4. But instead I have to count it off in my head, or rely on the formatted output.

jwwojak replied:
We have a disagreement then regarding these types of lists. That you can use 1, 1, 1, 1, 1 and have a nice sequenced list is an elegant feature. It's automating a tedious part of writing and allows us to focus on the instructions rather than on this code style (which is perfectly valid).

However, I can see that from a collaboration standpoint, it can be tricky, annoying, or require more effort to compare changes in a diff. I will try to stop, but have been writing lists using this method for so long it's now almost an autonomic process.  

Regarding the intent of Markdown itself, well, I'm sure we're breaking or bending all sorts of rules for this code throughout all our docs. I'll just end by quoting those Spanish colonial rulers in the New World who often replied to the monarch with, "obedezco pero no cumplo."

jwwojak replied:
Changed to numbered lists.  😢

ecormany commented on docs/flex-manual/docs/maintenance-service.md:
```diff
@@ -65,21 +65,23 @@ To clean the waste chute:
-5. Let the pipette air dry and reattach.
+1. Rinse off any remaining residue using a cloth dampened with distilled water.
+1. Let the pipette air dry and reattach.
+1. Reattach the pipette. During reattachment, you'll be prompted to recalibrate the pipette. This is optional, but recommended.
```
"reattach" in two consecutive steps. pick just one.

ecormany replied:
it looks like similar lists below put reattach in the step after air dry.

ecormany commented on docs/flex-manual/docs/maintenance-service.md:
```diff
@@ -223,55 +229,55 @@ Consult the [Flex IQ/OQ form](https://opentrons-flex-iq-oq-checklist.paperform.c
+    <tr>
+      <td>Yearly</td>
+      <td>Evaluate pipette performance</td>
+      <td>Evaluate the overall performance of your Flex pipettes. Replace any pipette that is not performing according to the <a href="pipette-specifications">published accuracy and precision standards</a>.</td>
```
Broken link. Leave as it was, as an autoref style link with the `markdown` attribute in the parent HTML tags.

The valid relative URL would be `../system-description/#pipette-specifications`, but when we refactor the Flex manual chapters, that link would break again. Autorefs avoids this as long as we keep the header name "Pipette specifications".

jwwojak replied:
Restored autoref style, but it is not rendering in the browser. Could be my branch doesn't have `autorefs` and this gets implemented later.

Also, adding an anchor to Preventative Maintenance `### Preventative maintenance { #preventative-maintenance }` because that future-proofs the link in case the section header text changes (autorefs use the header to make the ID).

ecormany reviewed (changes requested):
A couple small errors to fix, and one comment on future style (that requires no action now).

jwwojak committed:
Review commit

Fix ordered list numbering.

Break some cleaning steps apart:
- air dry
- reattach

Fix links to other docs using markdown autoref style

Added anchor to system-description.

Cleaned up some markdown tables.

ecormany commented on docs/flex-manual/docs/maintenance-service.md:
```diff
@@ -275,7 +302,7 @@ Along with routine cleaning, Opentrons also recommends the following optional pr
       <td>Yearly</td>
       <td>Evaluate pipette performance</td>
-      <td>Evaluate the overall performance of your Flex pipettes. Replace any pipette that is not performing according to the <a href="pipette-specifications">published accuracy and precision standards</a>.</td>
+      <td markdown>Evaluate the overall performance of your Flex pipettes. Replace any pipette that is not performing according to the [published accuracy and precision standards][Pipette specifications].</td>
```
OK, this is a little thornier than I thought. To get an autoref link (or any markdown link) working here, you have to specify the `markdown` attribute in all the parent tags. So `<table markdown>`, `<tbody markdown>`, `<tr markdown>`. But then you _cannot_ indent tags, because in markdown indentation indicates a code block. So that's pretty ugly.

The more elegant way is the HTML link, even though we may break it in the future. So be it, we will eventually add in some tests to catch such things!
```suggestion
      <td>Evaluate the overall performance of your Flex pipettes. Replace any pipette that is not performing according to the <a href="../system-description/#pipette-specifications">published accuracy and precision standards</a>.</td>
```

ecormany commented on docs/flex-manual/docs/system-description.md:
```diff
@@ -212,7 +212,7 @@ Locations of components of the 1-, 8-, and 96-channel pipettes.
 The pipettes pick up disposable plastic *tips* by pressing them onto the pipette *nozzles*, and then use the tips to aspirate and dispense liquids. The amount of total force required for pickup increases as more tips get picked up simultaneously. For smaller numbers of tips, the pipette attaches tips by pushing each pipette nozzle down into a tip. To achieve the necessary force to pick up a full  ...
 
-### Pipette specifications
+### Pipette specifications { #pipette-specifications }
```
This is a unique header so we don't need an anchor here (yet).
```suggestion
### Pipette specifications
```

ecormany reviewed (commented):
Last link fix to make. Sorry it took a few tries to arrive at the solution here!

jwwojak committed:
Update docs/flex-manual/docs/system-description.md

This header might have been throwing a warning during mkdocs serve. Can address it later if it comes up.

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak committed:
Update docs/flex-manual/docs/maintenance-service.md

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak committed:
Missing links in system-description.md

Found 2 instances of weirdness in migrated text and missing links.

Paragraph formatting, doesn't affect display but nice to fix up.

ecormany commented on docs/flex-manual/docs/system-description.md:
```diff
@@ -136,7 +136,7 @@ The primary user interface is the 7-inch LCD *touchscreen*, located on the front
 - Operation logs and error notifications
 
-For more information on using Flex via the touchscreen, see the of the Software and Operation chapter.
+For more information on using Flex via the touchscreen, see the [Touchscreen Operation section](software-operation.md#touchscreen-operation) of the Software and Operation chapter.
```
Good catch. Some links got disappeared in the initial conversion from PDF. I thought I'd found all of them!
